core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -159,13 +163,50 @@ final case class Nodepool(id: NodepoolLeoId,
                           autoscalingEnabled: Boolean,
                           autoscalingConfig: Option[AutoscalingConfig],
                           errors: List[KubernetesError],
-                          apps: List[App])
+                          apps: List[App],
+                          isDefault: Boolean)
 object KubernetesNameUtils {
   //UUID almost works for this use case, but kubernetes names must start with a-z
   def getUniqueName[A](apply: String => A): Either[Throwable, A] =
     KubernetesName.withValidation('k' + UUID.randomUUID().toString.toLowerCase.substring(1), apply)
 }
 
+case class DefaultNodepool(id: NodepoolLeoId,
+                           clusterId: KubernetesClusterLeoId,
+                           nodepoolName: NodepoolName,
+                           status: NodepoolStatus,
+                           auditInfo: AuditInfo,
+                           machineType: MachineTypeName,
+                           numNodes: NumNodes,
+                           autoscalingEnabled: Boolean,
+                           autoscalingConfig: Option[AutoscalingConfig]) {
+  def toNodepool(): Nodepool =
+    Nodepool(id,
+             clusterId,
+             nodepoolName,
+             status,
+             auditInfo,
+             machineType,
+             numNodes,
+             autoscalingEnabled,
+             autoscalingConfig,
+             List.empty,
+             List.empty,
+             true)
+}
+object DefaultNodepool {
+  def fromNodepool(n: Nodepool) =
```

Comment:
you can do something similar to https://github.com/DataBiosphere/leonardo/pull/1487/files#diff-35d0241c3de10d4f5cbec9c74325cb1aR46-R57